editor/webpack.config.js

```diff
@@ -324,13 +324,16 @@ const config = {
     : {},
 
   optimization: {
-    minimize: isProd,
-    minimizer: isProd
+    minimize: isProdOrStaging,
+    minimizer: isProdOrStaging
```

Comment:
I'm going to leave this change in for the merge so that we can smoke test on staging, and will then revert both lines back to `isProd` when we're satisfied that minifying hasn't broken anything.